x-pack/plugins/infra/public/hooks/use_kibana_space.ts

```diff
@@ -6,18 +6,21 @@
  */
 
 import useAsync from 'react-use/lib/useAsync';
-import type { Space } from '@kbn/spaces-plugin/public';
 import { useKibanaContextForPlugin } from './use_kibana';
 
 export type ActiveSpace =
   | { isLoading: true; error: undefined; space: undefined }
   | { isLoading: false; error: Error; space: undefined }
-  | { isLoading: false; error: undefined; space: Space };
+  | { isLoading: false; error: undefined; space: { id: string } | undefined };
```

Comment:
**question**: should the default returned space respect the `Space` type? It has 3 mandatory props `id`, `name` and `disabledFeatures`, you could update the default returned value to something like 
```
{
  id: 'default',
  name: 'Default',
  disabledFeatures: []
}
```
and leave space as the expected type

Reply:
Good idea, I wasn't sure about the import at the beginning but it should be fine. Changed ✅